Documentation/MRTK_Configuration_Dialog.md

```diff
@@ -64,12 +64,6 @@ More information can be found on the [Resonance Audio documentation](https://res
 
 ![UWP Settings](Images/ConfigurationDialog/ConfigurationDialogUWPSettings.png)
 
-### Enable MSBuild for Unity
```

Comment:
https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Tools/HolographicRemoting.html#legacy-xr-setup-instructions also has a lot of references to this tool. We have steps for using N4U, so I think we'll need to make those the main steps

Reply:
nice catch. thanks!

Reply:
i believe i have removed all of the relevant references.